searx/templates/simple/macros.html

```diff
@@ -23,8 +23,8 @@
   <span class="url_o{{loop.index}}"><span class="url_i{{loop.index}}">{{- part -}}</span></span>
   {%- endfor %}
   {{- result_close_link() -}}
-  {%- if result.img_src %}{{ result_open_link(result.url) }}<img class="image" src="{{ image_proxify(result.img_src) }}" title="{{ result.title|striptags }}">{{ result_close_link() }}{% endif -%}
-  {%- if result.thumbnail %}{{ result_open_link(result.url) }}<img class="thumbnail" src="{{ image_proxify(result.thumbnail) }}" title="{{ result.title|striptags }}">{{ result_close_link() }}{% endif -%}
+  {%- if result.img_src %}{{ result_open_link(result.url) }}<img class="image" src="{{ image_proxify(result.img_src) }}" title="{{ result.title|striptags }}" loading="lazy" width="200" height="200">{{ result_close_link() }}{% endif -%}
```

Comment:
The actual values are `width="101" height="101"` (with CSS).

![image](https://user-images.githubusercontent.com/1594191/145606073-86a7416d-0f6b-41ac-912b-8890c34a1c1a.png)

Why not see these value directly to avoid reflow?

Reply:
> The actual values are `width="101" height="101"` (with CSS).
> Suggestion: set the width & height to 101 to avoid reflow?

100 or 100,001 or ... it is all wrong (reflow?) since `rem` is related to the font size (em) of the browser and the font size is different on all devices.
 
IMO setting img sizes in em is the issue https://github.com/searxng/searxng/pull/573#discussion_r767320596

Should we set a img size in points / like we have done in the images template?